_includes/v22.2/ui/insights.md

```diff
@@ -0,0 +1,224 @@
+- **Transaction Execution**: The transaction fingerprint of the latest transaction execution.
+- **Insights**: The insight for the transaction execution.
+  - **High Contention**: The transaction execution experienced high [contention]({{ link_prefix }}performance-best-practices-overview.html#transaction-contention) time according to the threshold set in the [`sql.insights.latency_threshold` cluster setting]({{ link_prefix }}cluster-settings.html#setting-sql-insights-latency-threshold).
+- **Start Time (UTC)**: The timestamp when the transaction execution started.
+- **Contention Time**: The amount of time the transaction execution spent waiting in [contention]({{ link_prefix }}performance-best-practices-overview.html#transaction-contention).
+- **Application Name**: The name specified by the [`application_name` session setting]({{ link_prefix }}show-vars.html#supported-variables).
+
+### Transaction execution details
+
+The transaction execution details view provides more details on a transaction execution insight.
+
+{% if page.cloud != true -%}
+The following screenshot shows the execution details of the transaction execution in the preceding section:
+
+<img src="{{ 'images/v22.2/transaction_execution_details.png' | relative_url }}" alt="Transaction execution details" style="border:1px solid #eee;max-width:100%" />
+{% endif -%}
+
+The **Insights** column shows the name of the insight, in this case **High Contention**; the **Details** column provides details on the insight.
+
+- **Start Time (UTC)**: The time the transaction execution started.
+- **Transaction Fingerprint ID**: The transaction fingerprint ID of the transaction execution.
+
+#### Transaction with ID {transaction ID} waited on
+
+This section provides details of the transaction executions that block the transaction ID flagged with **High Contention**:
+
+- **Transaction Execution ID**: The execution ID of the blocking transaction execution.
+- **Transaction Fingerprint ID**: The transaction fingerprint ID of the blocking transaction execution.
+- **Transaction Execution**: The queries attempted in the transaction.
+- **Contention Start Time (UTC)**: The timestamp at which [contention]({{ link_prefix }}performance-best-practices-overview.html#transaction-contention) was detected for the transaction.
+- **Contention Time**: The time transactions with this execution ID was [in contention]({{ link_prefix }}performance-best-practices-overview.html#understanding-and-avoiding-transaction-contention) with other transactions within the specified time interval.
+- **Schema Name**: The name of the contended schema.
+- **Database Name**: The name of the contended database.
+- **Table Name**: The name of the contended table.
+- **Index Name**: The name of the contended index.
+
+### Statement Executions view
+
+The **Statement Executions** view provides an overview of all statement executions that have been flagged with insights.
```

Comment:
first mention of "statement executions" could link to https://www.cockroachlabs.com/docs/v22.2/show-statements.html to give add'l background, not sure if we'd want this link since it's a SQL shell thing tho